src/PerformanceTests/Common/Scenarios/SagaUpdate/SagaUpdateRunner.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System.Threading.Tasks;
+using NServiceBus;
+using NServiceBus.Logging;
+
+partial class SagaUpdateRunner
+    : BaseRunner
+{
+    public static bool Shutdown;
+    readonly ILog Log = LogManager.GetLogger(nameof(SagaUpdateRunner));
+
+    protected override async Task Start(ISession session)
+    {
+        var maxConcurrencyLevel = ConcurrencyLevelConverter.Convert(Permutation.ConcurrencyLevel);
+        var seedSize = maxConcurrencyLevel * 2;
```

Comment:
Have been considering to but the calculated concurrency value as a protected property in the base class. Not a real saver but now not each scenario needs to be responsible for knowing how to calculate it.

Same for the logic that drains the queue although I have been thinking about creating a behavior for that to very early in the pipeline.